job/dhcp.go

```diff
@@ -63,12 +63,15 @@ func (j Job) configureDHCP(rep, req *dhcp4.Packet) bool {
 			j.With("dhcp", isUEFI, "job", j.IsUEFI()).Info("uefi mismatch, using dhcp")
 		}
 
+		isSNP := dhcp.IsSNP(req)
+		j.Info("dhcp arch is x64 EFI, using SNP")
```

Comment:
this log should be behind an `if` check right ;)

Reply:
For sure.  I think I was going to put up inside, and then didn't because logger wasn't there and forgot to wrap it.  Fixed